### ISSUE TITLE
Primary_Text vs. Analyzed_Word

### ISSUE THREAD

**Issue by Glottotopia** [cldf/cldf#153]
One use case for IGT data stored in CLDF is the re-construction of nicely formatted interlinear text. For this, we need the source line, the IMT line, and the translation line. 

For the source line, for some use cases, we need the content of the column currently called `Analyzed_Word`. For other use cases, we need the content of the column called `Primary_Text`. For yet other use cases, either one would do. 

I have not completely grasped all of this, but I sense that there is some problem here in that `Primary_Text` and `Analyzed_Word` are both competing for the first line in a classical IGT three-liner. 

I open this issue to discuss the implications of this. 

Suppose you have a text in Thai, where each word consists of one morpheme. Then, the content of `Analyzed_Word` and `Primary_Text` will be exactly the same. Data providers might more or less randomly provide one or the other, but will not provide the information that the contents could be substituted. Data consumers might require `Analyzed_Word`-data, and might skip datasets where this information is not provided. It would be good if somehow data consumers could either be told a) that `Primary_Text` counts as `Analyzed_Word` for this data set, or that b) data providers are encouraged to duplicate their `Primary_Text` as  `Analyzed_Word` in these cases.

**Comment by LinguList** [comment]
Well, Thai is like Chinese, right? so you have still lots of compound words, not just morphemes, it is more the writing that makes this appear morpheme-syllabic, not the reality of the language itself? I Chinese texts, I force collaborators now to make explicit what is one word (also consisting of more than one character) and what is not, and they usually always agree, even for Old Chinese texts, where the majority is monosyllabic and people usually think it does not make sense.

**Comment by xrotwang** [comment]
As far as I understand, `Analyzed_Word` is mostly relevant together with `Gloss` (without `Gloss`, it might only add morpheme segmentation). So, whenever there's a `Gloss` and `Primary_Text` row, but no `Analyzed_Word`, the `Analyzed_Word` content should be automatically substituted by splitting `Primary_Text` on whitespace.

"Nicely formatted interlinear text" should never be reconstructed from `Primary_Text` and `Gloss`, because the `Primary_Text` column is not explicitly marked as list-valued, so cannot me "zipped" with the items in `Gloss` to create the aligned text.

**Comment by xrotwang** [comment]
There's no principle that says CLDF should be optimally normalized - in the sense of not containing redundant data. On the contrary, because ease of data reuse is the most important goal, it might be useful to provide de-normalized data. So, even if the content of `Primary_Text` and `Analyzed_Word` will be the same, both fields should be populated by data providers. (Semantically, they won't ever be the same because one is list-valued, the other is string valued.)

**Comment by LinguList** [comment]
We have some similar case in Value and Form in Wordlists. Value is in a bad case a very complex form with lots of brackets, in the ideal case, it is just the same as the Form.

**Comment by Glottotopia** [comment]
I agree about denormalization, redundancy and ease of reuse. Is this already stated somewhere? Could we add it to the guidelines somewhere?
